src/templates/healthcert/parsers/pdtHealthCertV1/generateMultiQrSection.tsx

```diff
@@ -46,6 +47,9 @@ export const generateMultiQrSection = (
     });
   }
 
+  const logoWidth = 130;
+  const logoHeight = logoWidth / 5;
```

Comment:
Is there any significance to the height being / 5? Will the width change dynamically?

Reply:
it is specified in the UI wireframe i believe, @witxhhaven

Reply:
This is because incidentally the ratio (width to height) of the tag is 5:1 . If the wordings in the tag were to change the ratio would need to change too. 

the height is width / 5 , is to help resizing the tag easier, just need to change the width value would do.